app/models/answer.rb

```diff
@@ -0,0 +1,6 @@
+class Answer < ActiveRecord::Base
+  belongs_to :survey
+  belongs_to :question
+  belongs_to :taker, class_name: "User"
+  belongs_to :choice
```

Comment:
Validations?

What prevents a user from answering twice? Or even choosing multiple choices for the same question?